clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy

```diff
@@ -179,6 +184,9 @@ class KubernetesApiAdaptor {
       return port
     }
 
+    containerDescription.livenessProbe = container?.livenessProbe ? fromProbe(container.livenessProbe) : null
```

Comment:
Could just have `fromProbe()` return null if the passed arg is null. But I can live with it this way if you'd rather not.